tensorflowonspark/pipeline.py

```diff
@@ -353,7 +358,15 @@ def _run_model(iterator, args):
     logging.info("outputs_tensor_info: {0}".format(outputs_tensor_info))
 
   result = []
-  with tf.Session(graph=ops_lib.Graph()) as sess:
+
+  global global_sess, global_input_mapping, global_output_mapping
+  if global_sess and global_input_mapping == args.input_mapping and global_output_mapping == args.output_mapping:
```

Comment:
Are you sure args.input_mapping and args.output_mapping are enough to ensure the session is reusable? We may want to add a check into args.export_dir, args.model_dir and args.tag_set.